packages/core/src/Stores/notification-store.js

```diff
@@ -623,9 +624,6 @@ export default class NotificationStore extends BaseStore {
                 } else {
                     this.removeNotificationMessageByKey({ key: this.client_notifications.dp2p?.key });
                 }
-                if (is_website_up && !has_trustpilot && daysSince(account_open_date) > 7) {
```

Comment:
Do we no longer need the condition daysSince(account_open_date) > 7? If that’s the case, how are we obtaining the account open date and verifying that the account is older than 7 days?

Reply:
We longer check for this condition for trustpilot notification, hence removed it.